src/Lykke.Job.TransactionHandler/Startup.cs

```diff
@@ -15,6 +15,10 @@
 using Lykke.Logs;
 using Lykke.SettingsReader;
 using Lykke.SlackNotification.AzureQueue;
+#if azurequeuesub
```

Comment:
azurequeuesub - this is parameter of the job template, it evaluates when `dotnet new lkejob` is executed. In the final application, all template #if's shouldn't be presented itself.

Reply:
Fixed.